frontend/src/e2e-test/pages/employee/units/unit-attendances-page.ts

```diff
@@ -340,6 +340,13 @@ export class UnitAttendancesPage {
   personCountSum(nth: number) {
     return this.page.findAllByDataQa('person-count-sum').nth(nth).innerText
   }
+
+  async assertAbsence(childId: UUID, date: LocalDate, type: string) {
+    await this.page
+      .findByDataQa(`absence-cell-${childId}-${date.formatIso()}`)
+      .find(`.absence-cell-right-${type}`)
+      .waitUntilVisible()
+  }
```

Comment:
`unit-diary-page.ts` already has an almost identical function. Could that code be reused to repurposed for this case?